draft-ietf-quic-http.md

```diff
@@ -557,12 +557,12 @@ MUST be treated as a connection error of type HTTP_MALFORMED_FRAME.
 
 When a PRIORITY frame claims to reference a request, the associated ID MUST
 identify a client-initiated bidirectional stream.  A server MUST treat receipt
-of PRIORITY frame with a Stream ID of any other type as a connection error of
-type HTTP_MALFORMED_FRAME.
+of a PRIORITY frame identifying a stream of any other type as a connection error
+of type HTTP_MALFORMED_FRAME.
 
-A PRIORITY frame that references a non-existent Push ID or a Placeholder ID
-greater than the server's limit MUST be treated as an HTTP_MALFORMED_FRAME
-error.
+A PRIORITY frame that references a non-existent Push ID, a Placeholder ID
+greater than the server's limit, or a Stream ID the client is not permitted to
```

Comment:
For stream IDs, is this "not **yet** permitted"?  The previous paragraph establishes the use of HTTP_MALFORMED_FRAME for the wrong type of stream.